packages/nextjs/pages/age-restricted-example/GenerateProof.tsx

```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
       <div>
         <h1 className="text-3xl font-bold">Step 2: Generating the proof ✅</h1>
         <p>
-          One of the reasons that Alice knows that she is not sharing her birth year with anyone is that the proof
-          generation is open source, and she herself can double check the code. Furthermore she can even generate the
-          proof herself locally. This is actually what we are doing in this implementation.
+          In order for <strong>Alice</strong> to really know that she&apos;s not sharing any private information with
+          the <strong>Ballon Store</strong> two things should be possible:
+          <ol>
+            <li>
+              1. The code that generates the the proof should be <strong>open source</strong> for Alice to review
+            </li>
+            <li>
+              2. The proof generation should happen in an environment trusted by<strong>Alice</strong> (ex: locally in
+              her laptop or phone)
+            </li>
+          </ol>
         </p>
         <p>
-          The circuit is located in{" "}
+          A circuit written in <strong>Noir</strong> is used for generating the proof and for generating a
+          Solidity-verifier. The <strong>Ballon Store</strong> will use the proof as input to execute the verifier
+          contract on-chain:{" "}
           <a href="https://github.com/Kryha/scaffold-eth-2-noir/blob/main/packages/noir/circuits/LessThanSignedAge/src/main.nr">
             <CodeText text="packages/noir/circuits/LessThanSignedAge/src/main.nr" />
           </a>
-          . It&apos;s written using Noir. It is the Noir-circuit that is used both for generating the proof and for
-          generating a Solidity-verifier that can help the balloon store verify the proof on-chain.
         </p>
         <p>
-          The proof is generated in the browser from the file{" "}
+          The proof is generated in the browser using the following libraries:
+          <ul>
+            <li>
+              * <CodeText text="aztec/bb.js" />
```

Comment:
```suggestion
              - <CodeText text="aztec/bb.js" />
```

So we don't mix it with the other asterix.